### PR TITLE
Remove TrailingCommaInMultilineArrayFixer

### PR DESCRIPTION
**Pull request by kubawerlos**

### CODE REVIEW

File: tests/Fixtures/Integration/misc/PHP7_4.test [hunk 1]
```diff
@@ -9,7 +9,7 @@ PHP 7.4 test.
     "no_null_property_initialization": true,
     "no_unset_on_property": true,
     "pow_to_exponentiation": true,
-    "trailing_comma_in_multiline_array": {"after_heredoc": true},
+    "trailing_comma_in_multiline": {"after_heredoc": true},
```
Comment (kubawerlos) [inline tests/Fixtures/Integration/misc/PHP7_4.test:12]: @keradus should this be also applied to version 2 line?

Comment (keradus) [reply tests/Fixtures/Integration/misc/PHP7_4.test:12]: would be nice, we don't need to use deprecations if not needed

File: UPGRADE-v3.md [hunk 2]
```diff
@@ -46,6 +46,7 @@ Old name | New name | Note
 `psr0`                                          | `psr_autoloading`                                                                 | use configuration `['dir' => x ]`
 `psr4`                                          | `psr_autoloading`                                                                 |
 `silenced_deprecation_error`                    | `error_suppression`                                                               |
+`trailing_comma_in_multiline_array`             | `trailing_comma_in_multiline`                                                     |
```
Comment (keradus) [inline UPGRADE-v3.md:49]: let's fill the 3rd column too

Comment (keradus) [inline UPGRADE-v3.md:49]:
```suggestion
`trailing_comma_in_multiline_array`             | `trailing_comma_in_multiline`                                                     | use configuration `['elements' => ['arrays']]`
```

File: tests/Fixtures/Integration/misc/PHP7_4.test (hunk 1 above)
Comment (kubawerlos) [reply tests/Fixtures/Integration/misc/PHP7_4.test:12]: done: https://github.com/FriendsOfPHP/PHP-CS-Fixer/pull/5672

**Comment by keradus** [comment]
Thank you @kubawerlos.

**Pull request merged**